app/src/main/res/values/strings.xml

```diff
@@ -220,88 +220,118 @@
          Below this string appears a list of locales for the user to choose as their preferred languages for web content. -->
     <string name="settings_language_choose_language_content_description">Web pages are sometimes offered in more than one language. Choose the languages you want to display in order of preference to the left side.</string>
 
-    <!-- This string is used to label a radio button in the 'Settings' Language dialog that, when pressed,
-         changes the app and the language of the speech-recognition-based search to 'English (US)'. -->
+    <!-- This string is used to label a radio button in the 'Settings' Voice Search and the Display Language dialogs that, when pressed,
+    changes the app and the language of the speech-recognition-based search to 'English (US)'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
```

Comment:
I'm confused. You mention that for the keyboard, only an "English" string would appear (as for Spanish and Norwegian). However, this string is "English (US)". So the comment doesn't seem quite right (shouldn't it be like the one for "English (UK)" below, and not call our keyboard?)
Am I missing something?

Reply:
For some reason the default keyboard string for English is "English US", so we don't really have a string for just English. Should we maybe remove the "US" from there?

Reply:
Maybe we could rather add a string just for the keyboard, that says "English"? Not sure that's possible, but would be great if we can.
I'm not a fan of removing the "US" since the localization can differ from en-UK

Reply:
I've added this as a new commit. "English" will be used for the keyboard related tags and "English (US)"/"English (UK) for the language related settings similarly to what we do with Norwegian. Let me know if that sounds reasonable to you.

Reply:
That sounds fine! thank you